azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -314,6 +330,17 @@ public void run() {
               lastExecutionDirCleanTime = currentTime;
             }
 
+            if (flowMaxRunningTimeInMins > 0 && currentTime - LONG_RUNNING_FLOW_KILLING_INTERVAL_MS > lastLongRunningFlowCleanTime) {
```

Comment:
@chengren311  is this configuration same for all flows. I was actually suggesting it to be passed during submission [#960]